libs/pavex_session_redis/src/lib.rs

```diff
@@ -0,0 +1,380 @@
+use anyhow::Context;
+use pavex::blueprint::{
+    Blueprint, config::ConfigType, constructor::Constructor, linter::Lint,
+    middleware::PostProcessingMiddleware,
+};
+use pavex::f;
+use pavex_session::{
+    SessionId,
+    store::{
+        SessionRecord, SessionRecordRef, SessionStorageBackend,
+        errors::{
+            ChangeIdError, CreateError, DeleteError, DeleteExpiredError, DuplicateIdError,
+            LoadError, UnknownIdError, UpdateError, UpdateTtlError,
+        },
+    },
+};
+use redis::{AsyncCommands, ExistenceCheck, SetExpiry, SetOptions, Value, aio::ConnectionManager};
+use serde;
+use std::num::NonZeroUsize;
+
+#[derive(Clone, Debug, Default, serde::Deserialize)]
+/// Configuration options for the Redis session store.
+pub struct RedisSessionStoreConfig {
+    /// Optional namespace prefix for Redis keys. When set, all session keys will be prefixed with this value.
+    #[serde(default)]
+    pub namespace: Option<String>,
```

Comment:
Let's include an example, showing the structure of a namespaced key (i.e. that `:` is used as separator).